src/main/java/org/junit/experimental/theories/ParameterSignature.java

```diff
@@ -34,7 +34,7 @@
         map.put(b, a);
     }
     
-    public static ArrayList<ParameterSignature> signatures(Method method) {
+    public static List<ParameterSignature> signatures(Method method) {
```

Comment:
I understand. I ask about some best practices like a "add deprecation with a comment about migration to `List` in next releases". Btw I put it back to the `ArrayList`